Formula/vtk.rb

```diff
@@ -79,6 +86,13 @@ def install
     inreplace Dir["#{lib}/cmake/**/vtkhdf5.cmake"].first,
               Formula["hdf5"].prefix.realpath,
               Formula["hdf5"].opt_prefix
+    # get rid of bad include paths on 10.14+
```

Comment:
Is this issue submitted upstream?

Reply:
It's not, but it doesn't seem to be present in 9.x but I think most formulae need changes to support 9.x. I'm not positive where precisely the actual root cause is, but this will allow pcl to build which has been broken in homebrew since boost was merged in a month ago.

Reply:
Could you start an upgrade to 9.x after this then? We should always provide the latest available version of software.

Reply:
Yeah I can start the upgrade process for it after this. The cleanup after the install of vtk in this PR really isn't ideal but I think it's worth getting out so we can upgrade `pcl` and may be affecting other formulae using vtk.